<p>This rule detects when a class defining a REST resource isn't documented</p>
<h2>Noncompliant Code Example</h2>
<pre>
@Path("clients")
public class ClientResource {
 ...
}
</pre>
<h2>Compliant Solution</h2>
<pre>
@Path("clients")
@Api("clients")
public class ClientResource {
 ...
}
</pre>
